openshift/olm/knative-serving.catalogsource.yaml

```diff
@@ -495,4 +2981,4 @@ spec:
   configMap: serverless-operator
   displayName: Serverless Operator
   publisher: Red Hat
-  sourceType: internal
\ No newline at end of file
+  sourceType: internal
```

Comment:
We can cut this file down but it needs manual change as I commented on https://github.com/openshift/knative-serving/pull/408#discussion_r390235448. So, perhaps placing this long file is OK?